lib/matplotlib/textpath.py

```diff
@@ -429,3 +430,23 @@ def _revalidate_path(self):
             self._cached_vertices = tr.transform(self._vertices)
             self._cached_vertices.flags.writeable = False
             self._invalid = False
+
+    def __deepcopy__(self, memo):
+        """Update path and create deep copy."""
+        self._revalidate_path()
+        cls = self.__class__
+        new_instance = cls.__new__(cls)
+        memo[id(self)] = new_instance
+        for k, v in self.__dict__.items():
+            setattr(new_instance, k, deepcopy(v, memo))
+        return new_instance
+
+    deepcopy = __deepcopy__
```

Comment:
Do you need this as a public method?  I'd say `copy.deepcopy(textpath)` is good enough?